spec/models/maths_and_physics/dqt_record_spec.rb

```diff
@@ -0,0 +1,54 @@
+require "rails_helper"
+
+RSpec.describe MathsAndPhysics::DQTRecord do
+  MATHS_JAC_CODES = %w[G100 G290 G310 G320 G900]
+  PHYSICS_JAC_CODES = %w[F300 F310 F320 F321 F331]
+  ELIGIBLE_JAC_CODES = MATHS_JAC_CODES + PHYSICS_JAC_CODES
+  NON_ELIGIBLE_JAC_CODES = %w[X100 L800 F100 C700 R100]
```

Comment:
Let's rename these to make it clearer that these are just examples, and not a definitive list:
```suggestion
  EXAMPLE_ELIGIBLE_JAC_CODES = MATHS_JAC_CODES + PHYSICS_JAC_CODES
  EXAMPLE_NON_ELIGIBLE_JAC_CODES = %w[X100 L800 F100 C700 R100]
```